director/jobs/models.py

```diff
@@ -337,6 +337,21 @@ class JobMethod(EnumChoice):
 
     sleep = "sleep"
 
+    @classmethod
+    def printable(cls, method: str) -> str:
+        """Convert the moethod into a printable string for the template."""
+        label = "Pulling"
+        label = "Pushing" if method == cls.pull.value else label
+        label = "Decoding" if method == cls.decode.value else label
+        label = "Converting" if method == cls.convert.value else label
+        label = "Compiling" if method == cls.compile.value else label
+        label = "Building" if method == cls.build.value else label
+        label = "Executing" if method == cls.execute.value else label
```

Comment:
These "ing" words imply that the job is running. What if it has failed or succeeded? IMO, just using the method name would be better.

Reply:
Yes, you're quite right. Without knowing the status, the tense can be misleading. I'll revert to using the name.

Reply:
I've actually removed this completely. There is additionally a new method on `Job` called `status_label` which will use the status where applicable or "In queue" etc. It will allow us to update this easily in the future.